src/Cron/MinutesField.php

```diff
@@ -16,8 +16,14 @@ public function increment(\DateTime $date, $invert = false)
     {
         if ($invert) {
             $date->modify('-1 minute');
+            if($date->format('i')=='59'){
+                $date->modify('+1 hour');
```

Comment:
Hmm, the cronline given above does work with just the hours class patched, though.

I can see the logic in the minutes class change, but I do not know enough about the internals to understand why that is causing the tests to fail.